### ISSUE TITLE
Unable to update test results generated in junit format from pytest in CircleCI

### ISSUE THREAD

**Issue by gjambaisivanandham** [gurock/trcli#232]
### TestRail CLI Version

v1.9.5

### CLI Environment

python:3.10.5

### TestRail Version

8.0.3 Default (3070)

### TestRail Instance Type

Enterprise Cloud

### Current behavior

My team is generating test results using https://www.npmjs.com/package/cypress-circleci-reporter, we had tests run in parallel and a lot of tests had the same name, I updated all of it and all the test and test suite have unique name, however while processing the test result, trcli is running into below issue. The below issue happens even when I removed the parallelization and running only a subset of tests.

```
TestRail CLI v1.9.5
Copyright 2024 Gurock Software GmbH - www.gurock.com
Parser Results Execution Parameters
> Report file: /tmp/glados/test_results/cypress/cypress-*.xml
> Config file: None
> TestRail instance: **************************** (user: *******************************)
> Project: Cypress Integration Tests
> Run title: Automated Cypress Tests from CircleCI workflow
> Update run: No
> Add to milestone: No
> Auto-create entities: True
Parsing JUnit report.
Processed 1 test cases in section cypress.
Processed 1 test cases in section cypress.
Processed 15 test cases in section cypress.
Checking project. Done.
Found 17 test cases not matching any TestRail case.
Error: Section duplicates detected in /tmp/glados/test_results/cypress/cypress-*.xml. This will result to failure to upload all cases.


Exited with code exit status 1
```

I checked the generated xml and there are no duplicate test case name but not sure why I am seeing this issue.

### Desired behavior

Process the results and update the test run

**Comment by MAlli-RSAC** [comment]
@gjambaisivanandham Were you able to resolve this issue? I'm running into a similar issue. Thanks!

**Comment by gjambaisivanandham** [comment]
> @gjambaisivanandham Were you able to resolve this issue? I'm running into a similar issue. Thanks!

Yes, I was able to resolve it. The issue was that trcli doesn't support parallel test runs and I was running the tests in parallel. To resolve I had to create a custom script to consolidate all the parallel test run results into one results file and then upload via trcli.

**Comment by MAlli-RSAC** [comment]
> > @gjambaisivanandham Were you able to resolve this issue? I'm running into a similar issue. Thanks!
> 
> Yes, I was able to resolve it. The issue was that trcli doesn't support parallel test runs and I was running the tests in parallel. To resolve I had to create a custom script to consolidate all the parallel test run results into one results file and then upload via trcli.

Interesting. Thank you. I'll see if I can do the same.

**Comment by MAlli-RSAC** [comment]
@gjambaisivanandham Yup, that worked! Thank you!
Now, I just have to figure out how to post the test case results in the consolidated script correctly, not just one test case.

**Comment by acuanico-tr-galt** [comment]
Thank you for reporting this issue. While there hasn’t been recent activity, please know that your feedback has been noted and is helping shape our internal improvements to TR CLI.
 
We’re closing this issue for now due to inactivity, but if this is still a concern or if you have additional details, feel free to reopen or open a new issue at any time.
